### ISSUE TITLE
Be consistent with request library used

### ISSUE THREAD

**Issue by Cruikshanks** [DEFRA/water-abstraction-team#8]
It's been spotted we are using ~~3~~ ~~**4**~~ **_5_** different types of HTTP request packages throughout the service.

- [request](https://www.npmjs.com/package/request)
- [request-promise](https://www.npmjs.com/package/request-promise)
- [request-promise-native](https://www.npmjs.com/package/request-promise-native)
- [got](https://www.npmjs.com/package/got)
- [axios](https://www.npmjs.com/package/axios)

On top of that **request**, **request-promise** and **request-promise-native** have been deprecated for more than 2 years 😩 😱 !

So, we need to move to one to be consistent and to deal with the fact we're using dead packages.

**Comment by Cruikshanks** [comment]
We already use **got** in WABS.

The Charging Module API uses **got**.

2 of the Water dev team still have to keep switching between the 2 projects making changes. Rather than have a big discussion and investigation about which package we should be using I believe we should just stick with **got**.

Fight me 👊 !

**Comment by jonathangoulding** [comment]
https://nodejs.org/en/blog/announcements/v18-release-announce/#fetch-experimental

**Comment by Cruikshanks** [comment]
When the current dev team first joined Water Abstraction we started creating issues for all the problems we saw. Whilst we were still getting to grips with the service it gave us opportunities to fix things and get stuck into the legacy code.

After a while, we became more confident and were able to get stuck into the critical feature at that time; SROC Supplementary billing. It also proved we could implement new and existing functionality in our [water-abstraction-system repo](https://github.com/DEFRA/water-abstraction-system) and integrate it with the existing apps. This, and the ongoing performance and stability issues with the legacy code have meant the whole delivery team are now onboard with us, over time, migrating _all_ functionality across.

So, as a dev team, we're instead going to focus our efforts on migrating functionality rather than fixing the legacy code. It means issues like this are now redundant as we don't intend to ever fix them.
